src/pages/how-to/retrieve.mdx

```diff
@@ -94,6 +94,14 @@ Replace `<YOUR_CID>`, `<FILE_NAME>`, and `<OUTPUT_FILE>` with their respective v
 | `<FILE_NAME>` | The _name_ of the file that you originally uploaded to Storacha. | `example.txt` |
 | `<OUTPUT_FILE>` | The path and filename that you want curl to save the file to. This can be different to `<FILE_NAME>`. | `./output-file.txt` |
 
+We support downdloading flat files out of the box, to download an extra large file or a directory, try passing `dag-scope=entity` in your request to not traverse recursively:
```

Comment:
```suggestion
We support downloading flat files out of the box. When downloading an extra large file or a directory, you may need to use `dag-scope=entity` in your request to avoid fetching the entire upload CAR:
```